samples/csharp_dotnetcore/08.suggested-actions/SuggestedActionsBot.cs

```diff
@@ -158,9 +160,9 @@ private static async Task SendSuggestedActionsAsync(ITurnContext turnContext, Ca
             {
                 Actions = new List<CardAction>()
                 {
-                    new CardAction() { Title = "Red", Type = ActionTypes.ImBack, Value = "Red" },
-                    new CardAction() { Title = "Yellow", Type = ActionTypes.ImBack, Value = "Yellow" },
-                    new CardAction() { Title = "Blue", Type = ActionTypes.ImBack, Value = "Blue" },
+                    new CardAction() { Title = "Red", Type = ActionTypes.ImBack, Value = "Red: ImBack ActionType" },
+                    new CardAction() { Title = "Yellow", Type = ActionTypes.ImBack, Value = "Yellow: ImBack ActionType" },
```

Comment:
I just re-read the PR description and it says ` users will be encourage to use MessageBack instead of ImBack and PostBack.` Should we then only use MessageBack for 3 card actions here?